.github/workflows/maven.yml

```diff
@@ -384,6 +384,48 @@ jobs:
             chmod +x ./mvnw
             ./mvnw -B -ntp clean package
 
+  Linux-riscv64-Build-JDK21:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        env:
+          cache-name: cache-m2-repository-${{ runner.os }}-jdk21-riscv64
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-pr-${{ env.cache-name }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-pr-${{ matrix.arch }}-pr-${{ env.cache-name }}-pr-
+      - uses: uraimo/run-on-arch-action@v2.4.0
+        name: Run commands
+        id: runcmd
+        with:
+          arch: riscv64
+          distro: ubuntu22.04
```

Comment:
Can we step down to 20.04 for wider compatibility across GLIBC variations?

Reply:
Done.

Reply:
Turns out JDK 21 from Adoptium has a dependency on newer Glibc 2.34. That's specific to JDK 21 though.